app/assets/javascripts/admin/addon/services/admin-emojis.js

```diff
@@ -1,52 +1,53 @@
-import Controller from "@ember/controller";
-import { action, computed } from "@ember/object";
-import { sort } from "@ember/object/computed";
-import { service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
```

Comment:
Nice to move this into a service instead of having it trapped in a controller :+1: